### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by harshitadatra**
nothing ..just trying to test initial commit

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Gautam2305) [inline index.html]: change your title to homepage

**Review by Gautam2305 — COMMENTED** [review]
suggested some changes

File: index.html (hunk 1 above)
Comment (harshitadatra) [reply index.html]: ok  i will do that

Comment (harshitadatra) [reply index.html]: did the changes..pls approve them

**Pull request merged**